torch/_dynamo/variables/builtin.py

```diff
@@ -1249,6 +1250,11 @@ def _unimplemented():
                 _unimplemented()
             return BaseListVariable.list_compare(tx, op, left, right)
 
+        if isinstance(left, SetVariable):
+            if not type(left) == type(right):  # Mismatch in BaseListVariable subclasses
+                _unimplemented()
+            return ConstantVariable(op(left._underlying_items, right._underlying_items))
```

Comment:
I'm worried this will be incorrect.  `==` on tensors does value comparison.  Does comparing a set of tensors even make sense?

What will happen for the other comparison operators?

Reply:
Yeah, I changed this to actually utilize real tensors, see below.

Reply:
As discussed, utilizing proxy nodes now :)